Stanislav_Kudin/07.Ansible.cruose/07.Ansible.cruise.md

```diff
@@ -0,0 +1,332 @@
+nginx : Install nginx for CentOS ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ 201.67s
+nginx : Install nginx for Ubuntu ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ 102.45s
+nginx : Add epel-release repo for CentOS ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 34.22s
+nginx : Create virtualhosts ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.89s
+nginx : Enable and start service nginx -------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.64s
+nginx : Checking the availability of virtual host sites --------------------------------------------------------------------------------------------------------------------------------------------------- 0.60s
+Gathering Facts ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.57s
+nginx : Create home directory for site -------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.53s
+nginx : Replace nginx.conf in CentOS ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.44s
+nginx : Copy first page on the virtualhost1 --------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.48s
+nginx : Create hosts file --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.47s
+nginx : Replace nginx.conf in Ubuntu ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.49s
+nginx : Copy first page on the virtualhost2 --------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.36s
+nginx : reload nginx -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.45s
+```
+

```

Comment:
Why so many empty lines ?

Reply:
May be when i put in vim. I don't know. ok i will fix. Thanks